### PR TITLE
added spanish language file

### PR DESCRIPTION
**Pull request by ALMerrill**
I added a spanish language file using the same emojis, but also added some alternate words that are used in spanish. there are several names for halloween, for example, but I added a few.

### CODE REVIEW

**Comment by ALMerrill** [comment]
I saw that another person added spanish which also had extra emojis. Mine has alternate words as well, so I guess we could merge both, but I also added a german one in another PR, as an extra

File: lang/es.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    {
+      name: 'fantasma',
+      emoji: '👻',
+    },
+    {
+      name: 'halloween',
+      emoji: '🎃',
+    },
+    {
+      name: 'noche de brujas',
```
Comment (LucasCtrl) [inline lang/es.js]: Sorry, at the moment multi-part words don't work (see issue #9)

File: lang/es.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+      emoji: '🎃',
+    },
+    {
+      name: 'dia de brujas',
```
Comment (LucasCtrl) [inline lang/es.js]: Sorry, at the moment multi-part words don't work (see issue #9)

**Review by LucasCtrl — CHANGES_REQUESTED** [review]

File: lang/es.js (hunk 1 above)
Comment (ALMerrill) [reply lang/es.js]: Fixed

File: lang/es.js (hunk 2 above)
Comment (ALMerrill) [reply lang/es.js]: Fixed

**Review by LucasCtrl — APPROVED** [review]

**Comment by LucasCtrl** [comment]
The bot will be updated during the night with the new translation file 😉

**Pull request merged**